integration_test/exqlite/ecto_sql/sandbox.exs

```diff
@@ -0,0 +1,279 @@
+defmodule Ecto.Integration.SandboxTest do
+  use ExUnit.Case
+
+  alias Ecto.Adapters.SQL.Sandbox
+  alias Ecto.Integration.{PoolRepo, TestRepo}
+  alias Ecto.Integration.Post
+
+  import ExUnit.CaptureLog
+
+  Application.put_env(:exqlite, __MODULE__.DynamicRepo, Application.get_env(:exqlite, TestRepo))
```

Comment:
there should be a more clever way of doing this. I tried adding to `test_helper` the following line:

```
Application.put_env(:ecto_sql, Ecto.Integration.SandboxTest.DynamicRepo, Application.get_env(:exqlite, TestRepo))
```

But that did not seem to work. So I forked it here for now.